src/Javinizer/Private/Scraper.Dmm.ps1

```diff
@@ -14,6 +14,28 @@ function Get-DmmContentId {
     }
 }
 
+function Get-DmmId {
+    param (
+        [Parameter(Mandatory = $true, Position = 0, ValueFromPipeline = $true)]
+        [Object]$Webrequest
+    )
+
+    process {
+        try {
+            # Expects ###ID##### or ID#####
+            $contentId = Get-DmmContentId $Webrequest
+            $Id = $contentId
+            $m = ($contentId | Select-String -Pattern '\d*([a-z]+)(\d+)$' -AllMatches).Matches
+            if($m.Groups.Count -gt 2 -and $m.Groups[1] -and $m.Groups[2]) {
+                $Id = $m.Groups[1].Value.ToUpper() + "-" + ($m.Groups[2].Value -replace '^0{1,5}', '').PadLeft(3, '0')
+            }

```

Comment:
The regex string `\d*([a-z]+)(\d+)$` needs to be updated to include ContentId's that include suffixes.
I'd say just add a `(.*)` to the end to match the suffix and add it to `$Id` in line 30. 

So new regex string of `\d*([a-z]+)(\d+)(.*)$`.

Reply:
Also, can you add a space in the between the `if` and the parentheses in line 29?